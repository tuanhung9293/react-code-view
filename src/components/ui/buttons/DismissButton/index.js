import React from 'react'
import IconButton from '../IconButton'
import dismiss from 'assets/dismiss.svg'

function DismissButton(props) {
  return <IconButton {...props}><img src={dismiss} alt="Dismiss" /></IconButton>
}

export default DismissButton