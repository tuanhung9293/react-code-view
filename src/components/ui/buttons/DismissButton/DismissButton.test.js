import React from 'react'
import {shallow} from 'enzyme'
import DismissButton from './'

it('renders as a DismissButton', () => {
  expect(shallow(
    <DismissButton />
  )).toMatchSnapshot()
})

it('should DismissButton pass props', () => {
  expect(shallow(
    <DismissButton dataTest="dataTest" />
  )).toMatchSnapshot()
})
