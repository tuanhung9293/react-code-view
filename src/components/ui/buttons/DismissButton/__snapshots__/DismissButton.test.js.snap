// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as a DismissButton 1`] = `
<IconButton>
  <img
    alt="Dismiss"
    src="test-file-stub"
  />
</IconButton>
`;

exports[`should DismissButton pass props 1`] = `
<IconButton
  dataTest="dataTest"
>
  <img
    alt="Dismiss"
    src="test-file-stub"
  />
</IconButton>
`;
